setup.py

```diff
@@ -184,6 +184,8 @@
         ],
     },
 
+    include_package_data=True,
+    package_data={'': ['/miniCran.R']},
```

Comment:
There shouldn't be a `/` before `miniCran.R`, which is causing the Window's tests to fail.